.ruby-version

```diff
@@ -1 +0,0 @@
-3.1.1
```

Comment:
No particular reason I suppose, although I was thinking that having this locked the version to a very specific version only in development. Since we intend to support this library for at least Ruby 2.6-3.2, I figured it'd be better to unlock the development version. Not strongly opinonated though

Reply:
I've re-added the file so the Ruby version is pinned in dev